docs/rfcs/rfc-007-subscriptions.md

```diff
@@ -281,67 +283,559 @@ An example of the full payload:
+            }
+            reviewers {
+                node {
+                    reputation
+                    ... on Person {
+                        reputation
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+-   _Deleted_ connection - **VALIDATE**
+
+```graphql
+subscription {
+    movieRelationshipDeleted(
+        where: {
+            movie: { title: "Matrix" }
+            createdRelationship: { actors: {}, directors: { Person: { node: { name: "Tom" } } } }
+        }
+    ) {
+        movie {
+            title
+        }
+        relationshipFieldName
+        deletedRelationship {
+            actors {
+                node {
+                    name
+                }
+            }
+            reviewers {
+                node {
+                    reputation
+                }
+            }
+        }
     }
 }
 ```
 
-**Filtering**  
+**Filtering**
+
 Filtering can be done with a `where` input field that will have the following properties
 
 -   `actor`: Filter of top level properties of the node (e.g. `actor`) from which the subscription is done
--   `relationshipName` can be filtered
 -   All of the `relationships` can be filtered either by related node or properties, by using the same properties as in the payload, only top-level properties of the related node can be filtered
--   `direction`
+-   By providing the relationship field name inside the `relationship` field, connection events of the type, properties and direction specified in the `@relationship` directive which the field is annotated with will be triggered.
+
+For instance, for subscribing only to connections of type `ACTED_IN` between types `Movie` and `Actor` as described in the `actors @relationship` annotation we can use the `actors` field:
+
+```graphql
+subscription {
+    movieConnected(where: { movie: { title: "The Matrix" }, relationship: { actors: {} } }) {
+        movie {
+            title
+        }
+        relationship {
+            actors {
+                node {
+                    name
+                }
+            }
+        }
+    }
+}
+```
 
-For instance, subscribing to only `actedIn` connections:
+If we are only interested in connections to `actors` that have a specific node or edge properties, we can further specify them:
 
 ```graphql
 subscription {
     movieConnected(
         where: {
-            relationshipName: "actors"
-            connectedMovie: { title: "The Matrix" }
-            relationship: { actors: { node: { name: "Keanu" } } }
+            movie: { title: "The Matrix" }
+            relationship: { actors: { node: { name_STARTS_WITH: "Keanu" }, edge: { screenTime_GT: 2345 } } }
         }
     ) {
-        connectedMovie {
+        movie {
             title
         }
         relationship {
             actors {
                 node {
                     name
                 }
+                edge {
+                    screenTime
+                }
```

Comment:
I would like to discuss this - whilst it is consistent with the filtering for Subscriptions, it is inconsistent with the rest of the API, which is a big thing we are trying to address at the moment. For instance, the filtering of Connection fields works the same as here, with `edge` and `node` at the same level - but then the data is returned in the format that I have suggested here. 

```suggestion
                screenTime
```

Reply:
That is correct!